sdv/metadata/table.py

```diff
@@ -222,6 +249,7 @@ def __init__(self, name=None, field_names=None, field_types=None, field_transfor
         self._entity_columns = entity_columns or []
         self._context_columns = context_columns or []
         self._constraints = constraints or []
```

Comment:
I suspect this line could be dropped now, and rely on the next on only.
I would also take care of the `None` possibility (so, the `constraints or []` part) inside the `_prepare` method, so it is hidden away.